src/utils.ts

```diff
@@ -50,6 +51,25 @@ export async function fileExists(filePath: string) {
   }
 }
 
+export async function cleanDir(dirPath: string) {
+  try {
+    const folderExists = await fs.stat(dirPath)
+    if (folderExists.isDirectory()) {
+      const files = await fs.readdir(dirPath)
+      await Promise.all(
+        files.map(async (file) => {
+          const filePath = path.join(dirPath, file)
+          await rimraf(filePath)
```

Comment:
can it remove the whole folder without traversal?

Reply:
If is recursive, why not use `fs.rm`?